### PR TITLE
updated tar package from 6.1.15 to 6.2.1

### PR DESCRIPTION
**Pull request by shivani-oh**
updated tar package from 6.1.15 to 6.2.1

### CODE REVIEW

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mderazon**
You can assign the PR to them by writing `/assign @mderazon` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fknowledgeowl-angular).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/knowledgeowl-angular/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by uniyal-vipin — APPROVED** [review]

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[uniyal-vipin](https://github.com/scheduleonce/knowledgeowl-angular/pull/145#pullrequestreview-2024892111 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mderazon**
You can assign the PR to them by writing `/assign @mderazon` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fknowledgeowl-angular).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/knowledgeowl-angular/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by oh-integrations** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *[uniyal-vipin](https://github.com/scheduleonce/knowledgeowl-angular/pull/145#pullrequestreview-2024892111 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fknowledgeowl-angular).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/knowledgeowl-angular/blob/qa/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
